app.js

```diff
@@ -38,7 +38,7 @@ hamburger.addEventListener('click', function () {
         each.style.textDecoration = 'none';
         each.style.color = 'white';
         each.style.fontWeight = '700';
-        each.style.fontSize = '2.5rem';
+        each.style.fontSize = '2.0rem';
        })
 
 
```

Comment:
## :red_circle: Your Pull Request Summary :red_circle:
![image](https://user-images.githubusercontent.com/93039359/215684670-51b306d3-b0e4-472f-9baf-9361173b70ba.png)
## :green_circle: Descriptive Pull Request Summary :green_circle:
![image](https://user-images.githubusercontent.com/93039359/215684883-64f24aed-5bcf-4dfe-b22d-ba61fcb70be6.png)

- [ ] Your pull request summary is not descriptive enough :fearful: Kindly make your pull request summary descriptive :+1: to make your pull request professional :+1: to show your professionalism :blush: